runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -149,6 +150,16 @@
     @Getter
     volatile CompletableFuture<Void> connectionFuture;
 
+    /**
+     * A future to a periodic tasks that sends pings
+     */
+    private ScheduledFuture<?> keepAliveTask;
+
+    /**
+     * The interval at which the keep-alive messages are sent
+     */
+    private long keepAliveInterval = 10;
```

Comment:
Should this be configurable? In the CorfuRuntimeParameters.

Reply:
Done.